Snowflake.Data.Tests/SFBaseTest.cs

```diff
@@ -30,8 +30,19 @@ public SFBaseTest()
         [OneTimeSetUp]
         public void SFTestSetup()
         {
-            string testConfigString = Encoding.UTF8.GetString(Properties.Resources.parameters);
+#if NET46

```

Comment:
Is there a #define or build target for 4.6.1 specifically, or can we assume everyone with 4.6 has been upgraded to 4.6.1?

Reply:
No tests build against .net 4.6.1 currently. I could add one.

Assumption is that if library a.k.a snowflake .net connector package is build against 4.6, so that every application using 4.6.1 should be able to use our connector right?

Reply:
I meant this as a general question about 4.6 versus 4.6.1. If memory serves, we (on the Azure team) used to build explicitly for 4.6.1 or greater, rather than for 4.6.